Makefile

```diff
@@ -310,3 +310,8 @@ lint-markdown:
 .PHONY: verify-readmes
 verify-readmes:
 	./verify_readmes.sh
+
+.PHONY: godoc
+godoc:
+	go install golang.org/x/pkgsite/cmd/pkgsite@latest
```

Comment:
This should be included in our tools instead of installed each invocation of `godoc`

Reply:
See description why I decided this way.

Reply:
There are commit releases. We should track those. Dependabot can handle updates for that as well.

Reply:
> Dependabot can handle updates for that as well.

Are you sure? The related issue and my experience says otherwise

Reply:
If not, I do not support adding this. It deviates from our tooling pattern and, as you pointed out, is not a stable set of tooling.

Reply:
I am totally fine with it. Closing the PR.